x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -142,20 +142,27 @@ export const ProcessEventDot = styled(
       const activeDescendantId = useSelector(selectors.uiActiveDescendantId);
       const selectedDescendantId = useSelector(selectors.uiSelectedDescendantId);
 
+      const logicalProcessNodeViewWidth = 360;
+      const logicalProcessNodeViewHeight = 120;
+      /**
+       * The `left` and `top` values represent the 'center' point of the process node.
+       * Since the view has content to the left and above the 'center' point, offset the
+       * position to accomodate for that. This aligns the logical center of the process node
+       * with the correct position on the map.
+       */
+      const processNodeViewXOffset = -0.172413 * logicalProcessNodeViewWidth * magFactorX;
+      const processNodeViewYOffset = -0.73684 * logicalProcessNodeViewHeight * magFactorX;
+
       const nodeViewportStyle = useMemo(
         () => ({
-          left: `${left}px`,
-          top: `${top}px`,
+          left: `${left + processNodeViewXOffset}px`,
+          top: `${top + processNodeViewYOffset}px`,
```

Comment:
:grey_question: It seemed a little more intuitive to me to keep the calculated value of `left` and `top` for the element aligned with the left and top but I guess it doesn't change anything material about the way it lays out, so if you like this better go for it.